entc/integration/customid/customid_test.go

```diff
@@ -116,6 +118,27 @@ func CustomID(t *testing.T, client *ent.Client) {
 	require.Equal(t, lnk.ID, blb.QueryLinks().OnlyX(ctx).ID)
 	require.Len(t, client.Blob.Query().IDsX(ctx), 3)
 
+	iSID := client.IntSID.Create().SaveX(ctx)
+	require.Equal(t, sid.ID("1"), iSID.ID)
+	iSIDChildID := sid.ID("100")
+	iSIDChild := client.IntSID.Create().SetID(iSIDChildID).SetParent(iSID).SaveX(ctx)
+	require.Equal(t, iSIDChildID, iSIDChild.ID)
+	require.Equal(t, iSID.ID, iSIDChild.QueryParent().OnlyX(ctx).ID)
+	iSIDBulk := make([]*ent.IntSIDCreate, 2)
+	iSIDBulk[0] = client.IntSID.Create().SetParent(iSID)
+	iSIDBulk[1] = client.IntSID.Create().SetParent(iSID)
+	iSIDChildren := client.IntSID.CreateBulk(iSIDBulk...).SaveX(ctx)
```

Comment:
```suggestion
	root := client.IntSID.Create().SaveX(ctx)
	require.EqualValues(t, "1", root.ID)
	cid := sid.ID("100")
	child := client.IntSID.Create().SetID(cid).SetParent(root).SaveX(ctx)
	require.Equal(t, cid, child.ID)
	require.Equal(t, rott.ID, child.QueryParent().OnlyX(ctx).ID)
	children := client.IntSID.CreateBulk(
		client.IntSID.Create().SetParent(root),
		client.IntSID.Create().SetParent(root)
	).SaveX(ctx)
```

Give these variables shorter names, please 🙏

Reply:
ok, but some variable names will conflict.
I'll wrap it by sub test.

in this way.
https://github.com/stoikheia/ent/pull/11/commits/1bfabcfb9621fd2fcdbb075e3d147d855b4f3e1e#

Reply:
I pushed this commit.
https://github.com/ent/ent/pull/2657/commits/5f5b11a835e234000fe40d6d857321a2e8b0ae5b
thanks